### PR TITLE
[MU4] Added navigation for palette panel popups

### PR DESCRIPTION
**Pull request by igorkorsukov**
Not very convenient requires discussion about directions of navigation (later)

### CODE REVIEW

File: src/instruments/qml/MuseScore/Instruments/InstrumentsPanel.qml [hunk 1]
```diff
@@ -242,6 +242,24 @@ Item {
                                 onVisibleChanged: {
                                     treeItemDelegateLoader.updateIsSelected()
                                 }
+
+                                property var contentYBackup: 0
+
+                                onPopupOpened: {
```
Comment (igorkorsukov) [inline src/instruments/qml/MuseScore/Instruments/InstrumentsPanel.qml:248]: Accidentally removed by prev commit, now restored

File: src/palette/qml/MuseScore/Palette/internal/MoreElementsPopup.qml [hunk 2]
```diff
@@ -53,13 +53,21 @@ StyledPopup {
     property bool drawGrid
 
     property int maxHeight: 400
-    implicitHeight: column.implicitHeight + topPadding + bottomPadding
-    width: parent.width
+    contentHeight: column.implicitHeight// + topPadding + bottomPadding
```
Comment (vpereverzev) [inline src/palette/qml/MuseScore/Palette/internal/MoreElementsPopup.qml:56]: any reason of this being commented?

Comment (igorkorsukov) [reply src/palette/qml/MuseScore/Palette/internal/MoreElementsPopup.qml]: removed

File: src/palette/qml/MuseScore/Palette/internal/TreePaletteHeader.qml [hunk 3]
```diff
@@ -147,78 +137,39 @@ Item {
         acceptedButtons: Qt.RightButton
 
         onClicked: {
-            toggleContextMenu()
+            menuLoader.toggleContextMenu(null)
```
Comment (vpereverzev) [inline src/palette/qml/MuseScore/Palette/internal/TreePaletteHeader.qml:140]: do we really need to call it if navigationParentControl is null? I'm not quite sure what would we expect in this case

Comment (igorkorsukov) [reply src/palette/qml/MuseScore/Palette/internal/TreePaletteHeader.qml:140]: this is clicked on RightButton, does not participate in navigation 
I generally think this action is not necessary

**Review by vpereverzev — APPROVED** [review]

**Pull request merged**